utils/utils.js

```diff
@@ -117,6 +117,13 @@ function getJSONTrimmed(context, url, writeKey, callback) {
   xhr.send();
 }
 
+function handleErrorWithBugsnag(error, breadcrumb) {
+  if (window.rsBugsnagClient) {
+    window.rsBugsnagClient.leaveBreadcrumb(breadcrumb);
```

Comment:
Why leaving breadcrumb is part of error notification?

Reply:
We can have it as a separate fn but again we have to check `window.rsBugsnagClient` is available or not. It is basically used to log the extra info w.r.t an error. We are adding the logger.error message or some custom message here. I didn’t feel like having another fn here.

Reply:
Breadcrumbs needn't be part of the error notification.

Reply:
?

Reply:
?